<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DelayFilterGUI</class>
 <widget class="QWidget" name="DelayFilterGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>66</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="delaySpinBox">
     <property name="suffix">
      <string> ms</string>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QLabel" name="delayLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Delay:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QDial" name="delayDial">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>66</height>
      </size>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="unitComboBox">
     <item>
      <property name="text">
       <string>Time</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Samples</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="3" rowspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
